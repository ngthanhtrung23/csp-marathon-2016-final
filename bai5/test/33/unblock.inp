11.23.
44423.
00.25.
6.775.
6.899.
AA8...
