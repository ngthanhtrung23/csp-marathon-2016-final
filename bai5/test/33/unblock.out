36
0 R 1
7 L 1
9 R 1
2 D 2
4 R 1
6 U 2
7 L 1
8 U 1
A R 3
8 D 1
7 R 1
6 D 3
0 L 1
4 L 1
2 U 1
7 L 1
8 U 2
9 L 3
A L 2
2 D 2
5 D 2
3 D 2
1 R 3
4 R 3
8 U 2
0 R 1
7 R 1
6 U 4
0 L 1
7 L 1
9 L 1
8 D 3
1 L 1
4 L 2
3 U 2
0 R 4

